utils/csgo_server.py

```diff
@@ -41,3 +44,12 @@ def make_available(self):
         self.score_message: discord.Message = None
         self.team_names: List[str] = None
         self.team_scores: List[int] = [0, 0]
+
+    def get_gotv(self) -> int:
+        if self.gotv is None:
+            status: str = valve.rcon.execute((self.server_address, self.server_port), self.RCON_password, 'status')
+            try:
+                self.gotv = status[status.index('port') + 5:status.index('port') + 10]
```

Comment:
Will this be automatically cast as an int?

Reply:
Yes, I tested it